src/payment/strategies/stripe-upe/stripe-upe-payment-strategy.ts

```diff
@@ -238,6 +215,82 @@ export default class StripeUPEPaymentStrategy implements PaymentStrategy {
         }
     }
 
+    private async _loadStripeElement(options: PaymentInitializeOptions) {
+        const { stripeupe, methodId, gatewayId } = options;
+
+        if (!stripeupe?.containerId) {
+            throw new NotInitializedError(NotInitializedErrorType.PaymentNotInitialized);
+        }
+
+        if (!gatewayId) {
+            throw new InvalidArgumentError('Unable to initialize payment because "gatewayId" argument is not provided.');
```

Comment:
Since you have done the check for gatewayId in line https://github.com/bigcommerce/checkout-sdk-js/pull/1385/files?diff=unified&w=0#diff-db8200b3b8110bda7e6a1017ce4388173a647e1455428b754b2861bd47d2e98dR52, does it make sense to pass the individual values as compared to the object?